apps/inviwodome/DomePreset.json

```diff
@@ -0,0 +1,98 @@
+{
+  "version": 4,
+  "cmakeMinimumRequired": { "major": 3, "minor": 23, "patch": 0 },
+  "configurePresets": [
+    {
+      "name": "dome-config",
+      "hidden" : true,
+
+      "cacheVariables": {
+        // Config
```

Comment:
comments are not part of the JSON standard

Reply:
Nopes but cmake is fine with them, so I don't care ;)